shop.html

```diff
@@ -0,0 +1,2093 @@
+                                </defs>
+                                </svg>
+                                                                                    
+                </div>
+            </form>
+          </div>
+        </div>
+      </nav>
+
+
+
+     
+
+
+      <div class="div">
+        <input type="checkbox">
+      </div>
+
+    <!-- /* labs */ -->
+    <div id="page2" class="container labCard ">
+        <div class="row  ">
+            <div class="col-12 text-center text-lg-start">
+                <h1 class="titleLab">Laptops</h1>
+                <h3 class="aboutLab">About 9,620 results (0.62 seconds)</h3>
+
+            </div>
+            <div class="col-12 col-lg-3  text-start  ">
+                <div class="list  d-flex flex-column  ">
+                    <ul class="d-inline-flex flex-column productList">
+                        <h2>Product Brand</h2>
+
+                        <li class="d-flex">
+                            <input type="checkbox" />
+                            <label>Asus</label>
+                        </li>
+
+                        <li class="d-flex">
+                            <input type="checkbox" />
+                            <label>Dell</label>
+                        </li>
+                        <li class="d-flex">
+                            <input type="checkbox" />
```

Comment:
All should be wrrapped inside of form the input type checkbix should be have name and value